README.rst

```diff
@@ -0,0 +1,86 @@
+xss-utils
+=============================
+
+|pypi-badge| |travis-badge| |codecov-badge| |doc-badge| |pyversions-badge|
+|license-badge|
+
+The ``README.rst`` file should start with a brief description of the repository,
+which sets it in the context of other repositories under the ``edx``
+organization. It should make clear where this fits in to the overall edX
+codebase.
+
+Utilities to prevent possible XSS attacks on Django/Mako templates.
+
+Overview (please modify)
+------------------------
+
+The ``README.rst`` file should then provide an overview of the code in this
+repository, including the main components and useful entry points for starting
+to understand the code in more detail.
+
+Documentation
+-------------
+
+The full documentation is at https://xss-utils.readthedocs.org.
```

Comment:
This is what I did for a different new repo:
```
The full documentation is in the docs directory.

TODO: Publish to https://xss-utils.readthedocs.org.
```

Reply:
Also:
1. The first time you mention XSS on this page, please use this instead `Cross Site Scripting (XSS)`.
2. Can you also add something like, `For more information, please read the Best Practices for Preventing XSS.` and point it to https://openedx.atlassian.net/wiki/spaces/SEC/pages/99090697/Best+Practices+for+Preventing+XSS